src/test/java/uk/gov/pay/api/it/validation/PaymentsResourceAmountValidationITest.java

```diff
@@ -28,7 +29,7 @@ public void createPayment_responseWith422_whenAmountIsNegative() throws IOExcept
                 "  \"amount\" : -123," +
                 "  \"reference\" : \"Some reference\"," +
                 "  \"description\" : \"Some description\"," +
-                "  \"return_url\" : \"http://somewhere.gov.uk/rainbow/1\"" +
+                "  \"return_url\" : \"https://somewhere.gov.uk/rainbow/1\"" +
```

Comment:
It was already like this, but probably we can use example.com, because we updated the docs as well - https://github.com/alphagov/pay-publicapi/commit/459919fbfc93bde7b5f4a9b286d85a9b2e5e7f98

Reply:
Meh, it’s only a test, doesn’t necessarily have to match the docs

I’m OK with using `something.gov.uk` domains where it probably would be a `gov.uk` domain (it’s an exception to my only-use-domain-names-reserved-in-RFC-2606-and-RFC-6761 rule)